.github/workflows/image.yaml

```diff
@@ -189,6 +189,8 @@ jobs:
          - flavor: "opensuse"
     steps:
     - uses: actions/checkout@v3
+      - run: |
```

Comment:
what about the release pipeline?

Reply:
@mudler for what I can tell both release.yaml and release-arm.yaml are ok, but I'm not entirely sure about release_bin.yaml does goreleaser somehow does any building too? if not then I think we're good